_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -19,32 +19,45 @@ practitioners with interests in the methodological, algorithmic, and software as
 [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
 we invite new contributors and those who have not met the core development team.
 
-## How do I attend?
+## Schedule
 
-To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
-will open in early 2023.
+A preliminary schedule is as follows. The full JuliaCon runs from July 25-29,
+and JuMP-dev specific events are scheduled on July 27-29.
 
-All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2023/coc/).
+#### July 25
+
+ * All day: JuliaCon workshops
+
+#### July 26
+
+ * All day: JuliaCon talks
+
+#### July 27
 
-## Travel support
+* All day: JuMP-dev talks
+* Evening: conference dinner
 
-To encourage people to attend JuMP-dev, we have funds to provide travel support
-to a limited number of applicants.
+#### July 28
 
-We particularly encourage people who have not been to a JuMP-dev before to
-apply.
+* All day: JuMP-dev talks
 
-[Apply for travel support](https://forms.gle/1pWBhu3VwBUophFV7)
+#### July 29
 
-The deadline for applications is January 31st.
+ * 09:00-14:00: JuMP-dev hackathon
+
+## How do I attend?
+
+To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
+will open in early 2023. Accepted speakers at JuMP-dev will receive a free
```

Comment:
we don't have an estimated date/month for tickets going on sale?

Reply:
Nope. I assume in the next couple of weeks. I think they need to co-ordinate with NumFOCUS, which might take a bit of time.